custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -47,13 +47,13 @@ def configure(self, params):
         sys.path.append(code_dir)
         self._model_adapter.load_custom_hooks()
 
-        super(PythonPredictor, self).configure(params)
-
         if to_bool(params.get("allow_dr_api_access")):
             logger.info("Initializing DataRobot Python client.")
             dr_api_endpoint = self._dr_api_url(endpoint=params["external_webserver_url"])
             dr.Client(token=params["api_token"], endpoint=dr_api_endpoint)
 
+        super(PythonPredictor, self).configure(params)
```

Comment:
Would it make sense, instead, to move the code block that initialises the client to the base language predictor?

Reply:
Sure, I think thats a good idea too.  I will make the change in the new PR.